mslib/_tests/test_thermolib.py

```diff
@@ -27,69 +27,66 @@
 
 import numpy as np
 import pytest
+from metpy.units import units
 
 import mslib.thermolib as tl
 
 
 def test_flightlevel2pressure2flightlevel():
-    fs = np.arange(1, 71000, 1000.) / 30.48
-    ps = tl.flightlevel2pressure_a(fs)
-    fs_p = tl.pressure2flightlevel_a(ps)
-    assert fs == pytest.approx(fs_p)
+    fs = np.arange(1, 71000, 1000.) / 30.48 * units.hft
```

Comment:
Probably better `(np.arange(...) * units.m).to(units.hft)`

Reply:
Done